<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainwindowClass</class>
 <widget class="QMainWindow" name="MainwindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1540</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mainwindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
				background-color:rgb(34,34,34)
			</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1536</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						background-color: rgb(31,31,31);
					</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>10</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Bahnschrift Condensed</family>
       <pointsize>25</pointsize>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
							QLabel {
							font-weight: 400;                   /* Normal weight */
							font: 25pt &quot;Bahnschrift Condensed&quot;; /* Font size and family */
							color: #ffffff;                     /* Font color (white) */
							}
						</string>
     </property>
     <property name="text">
      <string>Road-Map</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="Resource.qrc">:/img/img/pin.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton2">
     <property name="geometry">
      <rect>
       <x>1470</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset>
       <normaloff>:/newPrefix/Red_Cross_Mark_Icon_3d_Render_Illustration__Red_Cross__Cross_Mark__Cancelled_PNG_and_Vector_with_Transparent_Background_for_Free_Download-removebg-preview.png</normaloff>:/newPrefix/Red_Cross_Mark_Icon_3d_Render_Illustration__Red_Cross__Cross_Mark__Cancelled_PNG_and_Vector_with_Transparent_Background_for_Free_Download-removebg-preview.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>69</y>
       <width>1536</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(75, 75, 75)</string>
     </property>
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>1561</width>
      <height>791</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>451</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						font-size: 80px;
						letter-spacing: -.025em;
						font-weight: 400;
						font: 55pt &quot;Bahnschrift Condensed&quot;;
						color: rgb(255,255,255);
					</string>
    </property>
    <property name="text">
     <string>Create Graph </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>1150</x>
      <y>310</y>
      <width>241</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/newPrefix/map (1).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sliderbotton">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>80</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">
						background-color: rgb(21, 219, 201);
						border-radius: 10px;
						text-align: center;
					</string>
    </property>
    <property name="text">
     <string>&gt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="home" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>0</width>
      <height>781</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_city">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						color: rgb(255,255,255);
						font: 700 20pt &quot;Segoe UI&quot;;
						border-color:  rgb(255,255,255);
						border-radius: 10px;
						border-width: 1px;
						border-style: solid;
					</string>
    </property>
    <property name="text">
     <string>Add city</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Add_road">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>211</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						color: rgb(255,255,255);
						font: 700 20pt &quot;Segoe UI&quot;;
						border-color:  rgb(255,255,255);
						border-radius: 10px;
						border-width: 1px;
						border-style: solid;
					</string>
    </property>
    <property name="text">
     <string>Add road</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete_road">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>212</y>
      <width>191</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						color: rgb(255,255,255);
						font: 700 20pt &quot;Segoe UI&quot;;
						border-color:  rgb(255,255,255);
						border-radius: 10px;
						border-width: 1px;
						border-style: solid;
					</string>
    </property>
    <property name="text">
     <string>Delete road</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Delete_city">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>211</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						color: rgb(255,255,255);
						font: 700 20pt &quot;Segoe UI&quot;;
						border-color:  rgb(255,255,255);
						border-radius: 10px;
						border-width: 1px;
						border-style: solid;
					</string>
    </property>
    <property name="text">
     <string>Delete city</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>340</y>
      <width>861</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						border-color:  rgb(255,255,255);
						border-radius: 40px;
						border-width: 1px;
						border-style: solid;
					</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <widget class="QLineEdit" name="road_cost">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>240</y>
       <width>561</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							font-size: 18px;
							border-radius: 10px;
							border-width: 1px;
							border-style: solid;
							font-family: Medium, sans-serif;
							border-color:  rgb(34,34,34);
							background-color:rgb(255, 255, 255);
							color: rgb(34,34,34);
						</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>                                    Add road cost</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="city_name2">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>150</y>
       <width>561</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							font-size: 18px;
							border-radius: 10px;
							border-width: 1px;
							border-style: solid;
							font-family: Medium, sans-serif;
							border-color:  rgb(34,34,34);
							background-color:rgb(255, 255, 255);
							color: rgb(34,34,34);
						</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>                                    Add city name </string>
     </property>
    </widget>
    <widget class="QLineEdit" name="city_name1">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>561</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
							font-size: 18px;
							border-radius: 10px;
							border-width: 1px;
							border-style: solid;
							font-family: Medium, sans-serif;
							border-color:  rgb(34,34,34);
							background-color:rgb(255, 255, 255);
							color: rgb(34,34,34);
						</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>                                    Add city name </string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="Submit">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>710</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
						color: rgb(255,255,255);
						font: 700 20pt &quot;Segoe UI&quot;;
						border-color:  rgb(255,255,255);
						border-radius: 10px;
						border-width: 1px;
						border-style: solid;
					</string>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="0"/>
 <resources>
  <include location="Resource.qrc"/>
  <include location="QtApplication1.qrc"/>
 </resources>
 <connections/>
</ui>
